examples/js/postprocessing/MaskPass.js

```diff
@@ -25,11 +25,12 @@ THREE.MaskPass.prototype = {
 	render: function ( renderer, writeBuffer, readBuffer, delta, maskActive ) {
 
 		var context = renderer.context;
+		var state = renderer.state;
 
 		// don't update color or depth
 
-		context.colorMask( false, false, false, false );
-		context.depthMask( false );
+		state.setColorWrite( false );
+		state.setDepthWrite( false );
```

Comment:
It looks like using `context.colorMask` and `context.depthMask` create an inconsistent  state in three.js. I mean the internal WebGL state of three.js (maintained via `renderer.state`) can differ from the actual WebGL state if you directly use methods like `context.depthMask`. So a value like `currentDepthWrite` in `WebGLState` does not necessary reflect the actual state in WebGL. I think this is the reason why your mentioned example has problems with this PR...